sdk/modelsrepository/Azure.Iot.ModelsRepository/src/DependencyResolutionOption.cs

```diff
@@ -4,23 +4,24 @@
 namespace Azure.Iot.ModelsRepository
 {
     /// <summary>
-    /// The dependency processing options.
+    /// The model dependency resolution options.
     /// </summary>
     public enum DependencyResolutionOption
     {
         /// <summary>
-        /// Do not process external dependencies.
+        /// Try to get pre-computed model dependencies using .expanded.json.
+        /// If the model expanded form does not exist fall back to normal dependency processing.
         /// </summary>
-        Disabled,
+        TryFromExpanded,
```

Comment:
I preferred the previous order, in that it seemed to grow in functionality, and Disabled (0) was the default element.

Why the change?

Reply:
The change was from this feedback: https://github.com/Azure/azure-sdk-for-net/pull/19042#discussion_r583235811
And there are more implications like explicitly setting the enum integers since the processing logic depends on it.

I'll put it back in my upcoming PR.

Reply:
@azabbasi has a good point, but I think the new order is odd. I don't feel very strong about this, so I'll stay with just noting to you both that the order seems weird to me. If you both disagree, we can keep it as is. If you prefer to change it after reconsidering, that is fine too.

Reply:
I didn't know that Disabled is the default element. in the comment I mentioned that if TryFromExpanded is the default behavior, we should have it at the top.

Reply:
I'm not saying Disabled is the default element; it just made sense to me that Disabled should be the default element in an enum.

It sounds like the default for the ctor (if not specified) is expanded though, which is what you originally keyed off of.

Reply:
Let's move discussion to recently opened PR https://github.com/Azure/azure-sdk-for-net/pull/19072